src/Pages/License/Details/Cards/FeatureCard.jsx

```diff
@@ -3,15 +3,58 @@ import CustomCard from "Components/CustomCard/CustomCard";
 import CustomCardContent from "Components/CustomCard/CustomCardContent";
 import CustomCardHeader from "Components/CustomCard/CustomCardHeader";
 import CustomCardRow from "Components/CustomCard/CustomCardRow";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
 
+function createData(name, calories) {
+  return { name, calories };
+}
+
+const rows = [
+  createData("Frozen yoghurt", 159),
+  createData("Ice cream sandwich", 237),
+  createData("Eclair", 262),
+  createData("Cupcake", 305),
+  createData("Gingerbread", 356),
+  createData("Frozen yoghurt", 159),
+  createData("Ice cream sandwich", 237),
+  createData("Eclair", 262),
+  createData("Cupcake", 305),
+  createData("Gingerbread", 356),
+]; 
 const FeatureCard = ({ data }) => {
   return (
     <CustomCard>
       <CustomCardContent>
         <CustomCardHeader title="Features" />
         <Grid item container xs direction="column">
           <CustomCardRow title="Edition" value={data.edition} />
-          <Grid item>{/* TODO: Add table */}</Grid>
+          <Grid item>
+            <TableContainer component={Paper} sx={{height: 200}}>
+              <Table
+                sx={{ minWidth: 280}}
+                arialabel="simple table"
+              >
+                <TableBody>
+                  {rows.map((row) => (
+                    <TableRow
+                      key={row.name}
+                      sx={{ "&:lastchild td, &:lastchild th": { border: 0 } }}
```

Comment:
use makeStyles for styling instead of inline styles. See Login component for reference

Reply:
done